src/cryptonote_core/miner.cpp

```diff
@@ -209,6 +296,22 @@ namespace cryptonote
       {
         m_threads_total = command_line::get_arg(vm, arg_mining_threads);
       }
+      if(command_line::has_arg(vm, arg_bg_mining_enable))
+      {
+        set_is_background_mining_enabled( command_line::get_arg(vm, arg_bg_mining_enable) );
+      }
+      if(command_line::has_arg(vm, arg_bg_mining_idle_interval_seconds))
```

Comment:
Should the remaininder of these arguments be in a `if (is_background_mining_enabled())` block? And alternative, if background mining is disabled, should it verify that none of the arguments appear?